go.mod

```diff
@@ -137,6 +136,8 @@ replace (
 	k8s.io/metrics => k8s.io/metrics v0.24.2
 )
 
+require go.universe.tf/metallb v0.13.3
```

Comment:
I know I'm pedantic, but can you please put this requirement into the existing block of direct dependencies? :)) No idea why Go puts them into new `require` statements sometimes..